server/index.js

```diff
@@ -14,42 +14,72 @@ const port = 9000;
 app.use(cors());
 app.use(bodyParser.json());
 app.use(express.static('client/dist'));
+app.use('/:id', express.static('client/dist'));
 
-app.get('/api/products', (req, res) => {
-  controller.ikea_products.get(req, res, (err, data) => {
+
+// app.get('/api/products', (req, res) => {
+//   controller.ikea_products.get(req, res, (err, data) => {
+//     if (err) {
+//       // eslint-disable-next-line no-console
+//       console.error('error connecting to server');
+//       return;
+//     }
+//     // eslint-disable-next-line no-console
+//     console.log('ikea_products connected');
+//     res.send(data).status(200);
+//   });
+// });
+
+app.get('/api/products/', (req, res) => {
+  dbOperate.product_readALL((err, data) => {
     if (err) {
       // eslint-disable-next-line no-console
       console.error('error connecting to server');
       return;
     }
     // eslint-disable-next-line no-console
-    console.log('ikea_products connected');
     res.send(data).status(200);
   });
 });
 
-app.post('/api/products', (req, res) => {
+app.get('/api/products/:id', (req, res) => {
+  const idParam = req.params.id;
+  console.log(idParam);
+  res.send('Post request to /products received');
+});
+
```

Comment:
This doesn't appear to actually get any data from the db. Is it not finished yet?

Reply:
Not yet, still need to be finished.